roles/admin/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
All of these should include the "admin" tag so we can play through the entire role